components/Dialogs/tests/snapshot/MDCAlertControllerActionsTests.m

```diff
@@ -424,6 +424,13 @@ - (void)testVerticalActionsAreOrderedByEmphasis {
   view.orderVerticalActionsByEmphasis = YES;
 
   // Then
+  // Ensure snapshot view size matches actual runtime size of alert
+  MDCAlertControllerView *alertView = (MDCAlertControllerView *)self.alertController.view;
+  CGRect bounds = self.alertController.view.bounds;
+  CGSize size = [alertView calculatePreferredContentSizeForBounds:bounds.size];
+  bounds.size.height = size.height;
+  self.alertController.view.bounds = CGRectMake(0.f, 0.f, bounds.size.width, bounds.size.height);
```

Comment:
Is this logic typically what a client would need to do to get the correct sizing behavior? If not, can we adjust this to look more like the expected pattern of use?